topics/microbiome/tutorials/metaplasmidome_query/tutorial.md

```diff
@@ -758,7 +761,7 @@ Let's extract an overview of the annotations per metagenomic sequence by groupin
 >
 > 3. {% tool [Add Header](toolshed.g2.bx.psu.edu/repos/estrain/add_column_headers/add_column_headers/0.1.3) %} with the following parameters:
 >    - *"List of Column headers"*: `Metagenomic sequence name,Number of CDS,Number of annotated CDS,Number of associated KO,Number of associated PFAM`
->    - {% icon param-file %} *"Data File (tab-delimted)"*: output of **Select last** {% icon tool %}
+>    - {% icon param-file %} *"Data File (tab-delimited)"*: output of **Select last** {% icon tool %}
```

Comment:
I think we need to fix the upstream tool. 

Is this tool needed or can we replace it with something from IUC?

Reply:
This tool is quite nice for directly adding a header to a file without creating a new one, and then combining both files tail-to-head. Would you happen to have another suggestion to do the same?